app/Http/Controllers/TrainingDash.php

```diff
@@ -305,55 +305,6 @@ public function saveNewTicket(Request $request) {
             Log::info('Unable to send training ticket email: ' . $e);
         }
 
-        // Position type must match regex /^([A-Z]{2,3})(_([A-Z]{1,3}))?_(DEL|GND|TWR|APP|DEP|CTR)$/ to be accepted by VATUSA
```

Comment:
Please add this comment just above line 65 in the new UploadTrainingTickets.php. This information is undocumented and led to a multi-day troubleshooting session a couple of years ago. I put the information in the code to remind me of this requirement to prevent frustration in the future.